tox.ini

```diff
@@ -13,8 +9,17 @@ setenv =
   TOXENV={envname}
   XUNIT_FILE=nosetests-{envname}.xml
 commands =
-  {envbindir}/coverage run -p --omit="*tests*" --source=capone --branch \
-    setup.py test
+    {envbindir}/pytest \
+        --numprocesses 2 \
+        --cov=capone\
+        --cov-append \
+        --cov-report xml:coverage/coverage.xml \
+        --cov-report html:coverage/ \
+        --cov-fail-under 100 \
```

Comment:
This requires there be 100% coverage on every version of django or python. If that's true, then I guess that's fine, though it'll make it harder if we ever need conditional compatibility code.

Reply:
Yeah I agree that this may not be desired. For this PR, I was trying to fully reproduce the existing behavior: https://github.com/counsyl/capone/pull/7/files#diff-b67911656ef5d18c4ae36cb6741b7965L87

Reply:
The existing behavior was to generate a separate coverage file and combine them, and only then fail if the coverage is under 100%.

Reply:
Ah - I think this handled by [--cov-append](http://pytest-cov.readthedocs.io/en/latest/readme.html?highlight=cov-append#coverage-data-file) but I need to test a bit more to make sure it's doing exactly the same thing.

I believe even in the existing code that this coverage combining was not occurring across python versions since travis spins up a different container for 2.7 and 3.5